sympy/printing/lambdarepr.py

```diff
@@ -163,6 +163,14 @@ def _print_Min(self, expr):
     def _print_Max(self, expr):
         return '{0}(({1}))'.format('amax', ','.join(self._print(i) for i in expr.args))
 
+    def _print_DotProduct(self, expr):
+        arg1, arg2 = expr.args
+        if arg1.shape[0] != 1:
```

Comment:
numpy.dot does matrix multiplication, but I only want an actual dot product here. From other changes in this PR, DotProduct doesn't care about the shape of the input args, so long as they are both vectors of the same length.